source/_integrations/netdata.markdown

```diff
@@ -145,6 +145,7 @@ sensor:
     host: "192.168.1.2"
     port: "19999"
     name: SomeHostName
+    scan_interval: 00:00:10
```

Comment:
This is not an option for this integration, but platform configuration.

Therefore, it doesn't belong here.

Nevertheless, thanks for being willing to contribute 👍 

../Frenck